src/main/java/example/One.java

```diff
@@ -11,5 +11,8 @@ public void uncoveredMethod() {
     System.out.println(foo());
     System.out.println("Added new issue");
     System.out.println("This is new problem");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://127.0.0.1:9000/coding_rules#rule_key=squid%3AS106)